src/runtime/prte_finalize.c

```diff
@@ -74,7 +75,33 @@ int prte_finalize(void)
     /* release the cache */
     PRTE_RELEASE(prte_cache);
 
-    /* release the job hash table; pop the first element in key order and release it, repeat */
+    prte_display_prte_job_data("Finalize");
```

Comment:
What does this do? I can't find this function in my source

Reply:
Hah. Yeah that's my debug function. It shouldn't have been committed here. I'll remove it.

Reply:
Done.